infra/staging/main.tf

```diff
@@ -32,6 +32,11 @@ variable "db_password" {
   sensitive = true
 }
 
+variable "frontend_bucket_name" {
```

Comment:
Hm, relying on default values isn't great for reproducibility, so at some point I think we'd want something like `variables.tfvars`. Your change is in line with the existing code though, so let's not change it. We'll just keep this in mind as a TODO (or even better, we could leave a TODO comment).